CHANGELOG.md

```diff
@@ -5,6 +5,16 @@ All notable changes to Zebra are documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org).
 
+## Unreleased
+
+
+### Changed
+- Value pool "deferred" changes its identifier to "lockbox". `GetBlock` and `GetBlockchainInfo` 
+  RPCs returning value pools will now return the `lockbox` to the `FS_DEFERRED`
+  value pool to match [reference implementation](https://github.com/zcash/zcash/pull/6912/files#diff-decae4be02fb8a47ab4557fe74a9cb853bdfa3ec0fa1b515c0a1e5de91f4ad0bR276).
+  This only applies at the RPC layer. 
+- Slice `[GetBlockchainInfoBalance; 5]` type is aliased as `BlockchainValuePoolBalances`
```

Comment:
```suggestion
```

Reply:
Looks like too much detail for the main CHANGELOG. We need to add the breaking change here https://github.com/ZcashFoundation/zebra/blob/main/zebra-rpc/CHANGELOG.md under a `### Breaking changes` header. We can add this type alias change there under a `### Changed` header.